src/lib/serviceRoutines/putAllEntitiesWithTypeAndId.cpp

```diff
@@ -34,12 +34,26 @@
 #include "convenience/UpdateContextElementResponse.h"
 #include "convenienceMap/mapPutIndividualContextEntity.h"
 #include "serviceRoutines/putAllEntitiesWithTypeAndId.h"
+#include "serviceRoutines/postUpdateContext.h"
 
 
 
 /* ****************************************************************************
 *
 * putAllEntitiesWithTypeAndId - 
+*
+* PUT /v1/contextEntities/type/{entity::type}/id/{entity::id}
+*
+* Payload In:  UpdateContextElementRequest
+* Payload Out: UpdateContextElementResponse
+*
+* URI parameters:
+*   - attributesFormat=object
+*
```

Comment:
After 'not adding EntityId' in payload, I decided to not support those URI params.
I was wrong though, as entity::id and entity::type are set in the URL as well.

Coming up ...